### PR TITLE
Added 2 new algorythms (decrypt message, censor)

### PR DESCRIPTION
**Pull request by meg-1**
### Describe your change:

Added 2 new algorythms (decrypt message, censor)

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by IdoErel — CHANGES_REQUESTED** [review]
The comments are too long.
Keep the comments short and concise.

The code is implementing what's already been implemented in python.

Kind regards,
IdoErel

**Comment by bayrsaixan** [comment]
https://www.easeus.com/backup-software/?utm_source=youtube-heyiran-202107-tb-AsoftClick&utm_medium=referral

**Comment by meg-1** [comment]
> The code is implementing what's already been implemented in python.

Thank you a lot for the feedback! Quick question, you mean both algorithms have been implemented in Python already, right? Or just the censor algorithm?

**Review by xcodz-dot — CHANGES_REQUESTED** [review]
Thank you for trying but it seems that `decrypt` does not do any decryption but only removes the character provided to it and does not do any real world decryption. You might say that it does English deobfuscation but that is also false as well (in case you say so).

here is an example, both of them are equivalent, the latter being already inbuilt.
```python
decrypt("o$ne", "$") == "o$ne".replace("$", "")
```

and for the `censor` function that is also achieved by the same `replace`.

```python
censor("bad word", "bad") == "bad word".replace("bad", "***")
```

Okay, I get it that censor is still reasonable but `decrypt` is unnecessary.

**Comment by meg-1** [comment]
> Thank you for trying but it seems that `decrypt` does not do any decryption but only removes the character provided to it and does not do any real world decryption. You might say that it does English deobfuscation but that is also false as well (in case you say so).
> 
> here is an example, both of them are equivalent, the latter being already inbuilt.
> 
> ```python
> decrypt("o$ne", "$") == "o$ne".replace("$", "")
> ```
> 
> and for the `censor` function that is also achieved by the same `replace`.
> 
> ```python
> censor("bad word", "bad") == "bad word".replace("bad", "***")
> ```
> 
> Okay, I get it that censor is still reasonable but `decrypt` is unnecessary.
 
Thank you so much for the feedback, it's greately appreciated. Would it be okay if I remove the "decrypt" function, and only set a pull request for the censor function? Or does the censor function not hold any value to the repository, and it's not neccessary to commit?

**Comment by xcodz-dot** [comment]
As I am not a maintainer, I can only advise yes but please consider mentioning to any of the maintainers. My advise does not help with the merge decision of any PR and is completely up to maintainer

File: maths/average_welford.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+def average_welford(values):
```
Comment (xcodz-dot) [inline maths/average_welford.py:1]:
```suggestion
def average_welford(values) -> float:
```

**Review by xcodz-dot — CHANGES_REQUESTED** [review]

**Comment by meg-1** [comment]
@Kush1101 , hello! could you please review this pull request??

File: maths/average_welford.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+def average_welford(values: list) -> float:
+  """
+    >>> average_welford([1, 2, 3, 4, 5])
```
Comment (poyea) [inline maths/average_welford.py:3]: Please reformat the code to use 4 spaces as indentation

File: strings/decrypt.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+def decrypt(txt: str, symbol: str) -> str:
+  """
+  >>> decrypt("o$ne", "$")
+  one
```
Comment (poyea) [inline strings/decrypt.py:4]: 4 spaces too.

**Comment by poyea** [comment]
Sometimes the CI is not running - just LMK if that's the case

**Comment by meg-1** [comment]
> Sometimes the CI is not running - just LMK if that's the case

ty, i think that might have been the case. will definitely inform if it happens again!

**Pull request closed without merging**